### ISSUE TITLE
add "related:12345" powertag which email notifies author of related post

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#646]
Also, `response:12345` should also notify the author of the original post. See [this page](https://publiclab.org/wiki/power-tags#Automated+power+tags,+not+for+manual+adding) for more on Power Tags, but basically adding `response:12345` where 12345 is the `nid` of a node, makes it appear in the sidebar of both nodes. Examples: https://publiclab.org/tag/response:*

Could be done in `DrupalTag:subscribers` ([here](https://github.com/publiclab/plots2/blob/master/app/models/drupal_tag.rb#L141-L164)) or in the `SubscriptionMailer`: https://github.com/publiclab/plots2/blob/master/app/mailers/subscription_mailer.rb

**Comment by ananyo2012** [comment]
Ok this should be an easy fix. So you wan't an email notification on note creation if that tag is present? A new mailer view and action should do it.

**Comment by jywarren** [comment]
Yep! And we should consider how to display related notes. Maybe same place
as "response" notes?

**Comment by ananyo2012** [comment]
Yes we can just make one more section for displaying it probably we can call it "Mentioned notes"

**Comment by jywarren** [comment]
Well, the related tag could be in the body of the post or just in the tag
field, so maybe we should just say "related notes".

**Comment by ananyo2012** [comment]
But there is already a term called "Related notes" for notes tagged with same tags.

**Comment by jywarren** [comment]
It's true -- but do you think `related:12345` should then be referred to as
"mentioned" instead, somehow? OK how about we rename the current same-tags
list as "Similar notes" and the new section will be "Related notes"?

**Comment by ananyo2012** [comment]
@jywarren Yes we can definitely do that.

**Comment by gauravano** [comment]
@jywarren should we re-open this as a new issue? Thanks!

**Comment by jywarren** [comment]
Yes but I think we can use the current `response:00000` tags to accomplish
it! And we could build this into the existing "notify tag followers"
mailer. Thanks!

**Comment by NitinBhasneria** [comment]
@jywarren Is this issue still pending.
